packages/rest-express/src/transport-express.ts

```diff
@@ -0,0 +1,247 @@
+import AccountsError from '@accounts/error'
+
+import AccountsServer from '@accounts/server';
+
+import { Configuration } from './types/configuration';
+
+import { Router, Request, Response, NextFunction } from 'express';
+
+import { getConnectionInformations } from './utils/get-connection-informations';
+
+interface RequestWithSession extends Request {
+  session: { [key: string]: any };
+}
+
+export default class TransportExpress {
```

Comment:
I suppose, but I think we should probably design them before creating the interface, just in case we need to change the Express Transport too
What do you think ?